scripts/api-ensure-no-diff.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
Could you put a high level comment for what this script does?

What do you think of naming this script: `ensure-protos-compiled.sh`?

Reply:
done.